.github/workflows/workflow-roadmap.yml

```diff
@@ -27,14 +27,34 @@ concurrency:
 jobs:
   update-workflow-issue:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        label: ["jvm"] # add if desired: "codebase-health", "core", "infrastructure", "go", "security", "dx"
```

Comment:
cc @TBD54566975/ftl-team if you want to automate your roadmap dashboards, you can opt in here

Reply:
@worstell the `dx` board is ready to go if you want use us as another test team for this :)